docs: What’s new & Upgrade guide 10.2

imatwawana opened the pull request:
This PR adds the _What’s new_, which describes new features and changes available in Grafana v10.2, and the v10.2 _Upgrade guide_.

imatwawana committed:
Added 10.2 what's new and upgrade guide

imatwawana committed:
Fixed title and added guidance

eleijonmarck committed:
add: no basic role feature to whats new

vtorosyan commented:
@eleijonmarck thanks for adding the section about what's new. As an FYI, for the next time, instead of direct commit to this PR, please create a separate PR against this branch, so that there is a chance to do a separate review first, before merging. 

For now I think the best would be to review the change here, instead of reverting the commit and creating a new PR.

vtorosyan commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,52 @@
+### No Basic Role
+Generally available in all editions of Grafana
+
+Introducing a new basic role with no permissions, called "No Basic Role." This role is perfect for when users would like their service accounts or users to have tailored customized RBAC permissions. This can be set as a basic role via the API or UI.
```
@eleijonmarck couple of thoughts:

- I think readers would benefit from having references to some of the definitions, such as what is a basic role and a permission. 
- If the feature is GA, there should not be a feature toggle. Can you please check and either remove the toggle, or update to the correct availability note?
- For the what's new content, might be useful to have a note around what was previously and what users have now. 
- I'd remove the note about what is important to remember, and link to the respective docs instead. 


Also, would appreciate if you can send a new PR with the changes, instead of changing directly here. Thank you!

eleijonmarck commented:
@it's easy to revert the commit and make a PR

eleijonmarck committed:
Revert "add: no basic role feature to whats new"

This reverts commit eb2e31c3dcf013c45b4102f3a149826f5d4c7fe5.

oscarkilhed committed:
Add field min/max calculation to what's new

mgyongyosi committed:
refresh token handling

mgyongyosi committed:
update

oscarkilhed committed:
Remove whitespace

oscarkilhed commented:
Oh, sorry, seems like I also misunderstood the process here. Will revert and make a PR

oscarkilhed committed:
Revert "Add field min/max calculation to what's new"

This reverts commit 9b66ddd0ecfed3b9c3b1dee9cdac6d748e24cffa.

mgyongyosi committed:
Revert refresh token handling and move it to separate PR

mgyongyosi committed:
Docs: Add permission validation enabled by default to what's new (#76376)

* Add permission validation enabled by default to whats new

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* update

* Apply suggestions from code review

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Fix Grafana editions note

* run prettier

---------

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,69 @@
+
+<!-- Mihaly Gyongyosi -->
+
+_Available in public preview in all Grafana Open Source and Enterprise_
```
```suggestion
_Available in public preview in Grafana Open Source and Enterprise_
```

imatwawana committed:
Fixed typo

eleijonmarck committed:
Docs: Add No basic role feature description (#76345)

* add whats next for cloud

* Update docs/sources/whatsnew/whats-new-next/index.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-next/index.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-next/index.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-next/index.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-next/index.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-next/index.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

---------

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

Jguer committed:
Auth: Whats new v10.2 (#76369)

* add whatsnew auth

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

* Apply suggestions from code review

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* separate ds and db managed permissions whats new

* Apply suggestions from code review

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

---------

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

oscarkilhed committed:
What's new in transformations 10.2 (#76430)

* first draft

* spelling

* Add doc links and images

* Fix relrefs

* Apply suggestions from code review

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Add information about availability

* update availability for string format transform

* add field from calc modifications

* Fixed link version syntax

* Apply suggestions from code review

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

---------

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>
Co-authored-by: Victor Marin <victor.marin@grafana.com>

oscarkilhed committed:
Add field min/max calculation to what's new (#76401)

* Add field min/max calculation to what's new

* fix whitespace

* Fix which editions it's available in

* Apply suggestions from code review

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* change availability

* reduce image size

---------

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

katebrenner commented on docs/sources/whatsnew/_index.md:
```diff
@@ -76,6 +76,7 @@ For a complete list of every change, with links to pull requests and related iss
 
 ## Grafana 10
 
+- [What's new in 10.1]({{< relref "whats-new-in-v10-2/" >}})
```
```suggestion
- [What's new in 10.2]({{< relref "whats-new-in-v10-2/" >}})
```

imatwawana committed:
Removed What's new next addition from this PR

oscarkilhed commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,189 @@
+
+With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/cust ...
+
+<<<<<<< HEAD
```
```suggestion
```

imatwawana committed:
Added alertmanager update (#76584)

* Added alertmanager update

* Fixed order of items

* Added availability info

imatwawana committed:
remove typo

Co-authored-by: Oscar Kilhed <oscar.kilhed@grafana.com>

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,196 @@
+{{< figure src="/media/docs/grafana/panels-visualizations/localminmax.png" max-width="300px" caption="Stat panel visualization with min/max calculated per field" >}}
+In this example, using the same data, with the min and max calculated for each individual field, we get a much better understanding of how the current value relates to the historical values. The A-series no longer exceeds the 10% threshold; in fact, it's now clear that it's at a historical low.
+
+# This is not only useful in the stat visualization; gauge, bar gauge, and status history visualizations, table cells formatted by thresholds, and gauge table cells all benefit from this addition.
```
```suggestion
This is not only useful in the stat visualization; gauge, bar gauge, and status history visualizations, table cells formatted by thresholds, and gauge table cells all benefit from this addition.
```

imatwawana committed:
Remove heading formatting

nmarrs committed:
Merge remote-tracking branch 'origin' into docs/whats-new-v10.2

nmarrs committed:
initial draft of dataviz squad deliverables for 10.2 whats new

nmarrs committed:
add dashgpt (#dashboard-ai) whats new draft

vtorosyan commented:
@imatwawana sorry I am a bit confused. I see that [this commit](https://github.com/grafana/grafana/pull/75909/commits/ad7cf90b868029de5d2a8cb839f11d985c6a30e8) removed the information about introducing a new feature (No basic role) which is available both in Cloud and on prem. I can't see the what's new in any other place, just want to make sure we won't miss it out.

ifrost committed:
What's New: Correlations in 10.2 (#76505)

* What's new about correlations

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update availability

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v10-2.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

---------

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

imatwawana commented:
> @imatwawana sorry I am a bit confused. I see that [this commit](https://github.com/grafana/grafana/pull/75909/commits/ad7cf90b868029de5d2a8cb839f11d985c6a30e8) removed the information about introducing a new feature (No basic role) which is available both in Cloud and on prem. I can't see the what's new in any other place, just want to make sure we won't miss it out.

Yeah, sorry the process this time around is a bit weird. I'll be adding it later. It will ultimately exist in both places.

imatwawana committed:
Updated linking guidance in template and fixed links

imatwawana committed:
Fixed link

Co-authored-by: Kate Brenner <32871890+katebrenner@users.noreply.github.com>

imatwawana committed:
Docs: update whats new links (#76639)

Replaced relrefs with fully qualified URLs

nmarrs committed:
DashGPT: update availability status to public preview

nmarrs committed:
Merge remote-tracking branch 'origin' into docs/whats-new-v10.2

codeincarnate committed:
Update What's New to Include information on Timezone and Applicability changes in transformations (#76571)

* Update what's new

* Fix misspelling

* Update to fix wording

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Remove merge conflict flag

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Fix capitalization

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Fix wording

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Sentence casing

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Fix up wording

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Fix availability

* One last fix :)

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

* Fixed typo (I introduced!)

---------

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

imatwawana committed:
Docs: add Cloud what's new items to 10.2 what's new (#76642)

* Added Cloud what's new content

* Apply suggestions from code review

Co-authored-by: Jennifer Villa <jvilla2013@gmail.com>

* Updated link formatting

* Updated Browse Dashboards with the latest content

* Updated availability

* Updated wording

Co-authored-by: Haris Rozajac <58232930+harisrozajac@users.noreply.github.com>

* Fixed typo

Co-authored-by: Haris Rozajac <58232930+harisrozajac@users.noreply.github.com>

---------

Co-authored-by: Jennifer Villa <jvilla2013@gmail.com>
Co-authored-by: joshhunt <josh@trtr.co>
Co-authored-by: Haris Rozajac <58232930+harisrozajac@users.noreply.github.com>

imatwawana committed:
Fixed title

imatwawana committed:
Removed all editions language

imatwawana committed:
General edits

nmarrs commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,461 @@
+### Generative AI features for dashboards
+
+// Is this actually generally available given steps of needing to also install LLM plugin / have openAI key?
+_Generally available in of Grafana Open Source and Enterprise_
```
This should be public preview status 😬

imatwawana replied:
Yep, I got that from the Slack thread. It's in my local set of updates.

imatwawana committed:
Copy edits

imatwawana committed:
Fixed availability note

imatwawana committed:
Copy editing

Removed Visualize enum entry

Added sharing section

Added alerting entry

Added introduction

Made minor copy edits

imatwawana committed:
Fixed capitalization

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,461 @@
+
+<!-- Jo Guerreiro -->
+
+_Generally vailable in Grafana Enterprise_
```
```suggestion
_Generally available in Grafana Enterprise_
```

imatwawana committed:
Fixed typo

nmarrs committed:
Merge remote-tracking branch 'origin' into docs/whats-new-v10.2

mjseaman committed:
add CTA and gif to AI features

mjseaman committed:
update recorded queries text

mjseaman committed:
Update edition availability

Cloud permissions were missing (even though this is an OSS release I think it's important to include details about Cloud availability) and some features listed the wrong availability.

mjseaman committed:
reorder topics for emphasis

Emphasize the most exciting improvements

mjseaman committed:
add links to feature toggle docs

mjseaman committed:
move recorded queries

mjseaman committed:
update wording and titles for readability

mjseaman committed:
add visualization media

mjseaman committed:
update wording in dataviz section

nmarrs committed:
dashgpt: finalize whats new for 10.2 (finally) 😬

nmarrs committed:
Merge remote-tracking branch 'origin' into docs/whats-new-v10.2

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+### Tempo data source
+
+We've placed special focus on the Tempo data source over the past couple of months with new features, query performance improvements, and a better query exerience.
```
```suggestion
We've placed special focus on the Tempo data source over the past couple of months with new features, query performance improvements, and a better query experience.
```

imatwawana committed:
Fixed typo

nmarrs committed:
DataViz: finalize whats new content 😬

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+<!-- Thanos Karachalios -->
+
+_Generally Available in all editions of Grafana_
```
```suggestion
_Generally available in all editions of Grafana_
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+## Correlations
+
+Grafana Correlations is a new Public Preview feature you can use to establish links from any data source query to any other, carrying forward data like namespace, host, or label values. This is extremely powerful for performing root cause analysis with a diverse set of data sources. For more information, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administratio ...
```
```suggestion
Grafana Correlations is a new public preview feature you can use to establish links from any data source query to any other, carrying forward data like namespace, host, or label values. This is extremely powerful for performing root cause analysis with a diverse set of data sources. For more information, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/correlations/).
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+<!-- Kristina Durivage -->
+
+_Available in public preview in Grafana Open Source and Enterprise_
```
```suggestion
_Available in public preview in all editions of Grafana_
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+<!-- Piotr Jamróz -->
+
+_Available in public preview in Grafana Open Source and Enterprise_
```
```suggestion
_Available in public preview in all editions of Grafana_
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+### Use AI to generate titles, descriptions, and change summaries
+
+_Available in public preview in Grafana Open Source and Enterprise_
```
```suggestion
_Available in public preview in all editions of Grafana_
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+{{< figure src="/media/docs/grafana/dashboards/auto-generate-description-10-2.gif" max-width="750px" caption="Auto-generate a panel description using AI" >}}
+
+### Find your dashboard faster with the new Dashboard Browse screen
```
```suggestion
### Find your dashboard faster with the new Dashboards browse screen
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+_Generally available in all editions of Grafana_
+
+The new Browse screen for dashboards features a more compact design, making it easier to navigate, search for, and manage your folders and dashboards. The new interface also has many performance improvements, especially for instances with a large number of folders and dashboards.
```
```suggestion
The new browse screen for dashboards features a more compact design, making it easier to navigate, search for, and manage your folders and dashboards. The new interface also has many performance improvements, especially for instances with a large number of folders and dashboards.
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+To make using folders easier and more consistent, there's no longer a special **General** folder. Dashboards without a folder, or dashboards previously in the **General** folder, are now shown at the root level.
+
+To try it out, head to the Dashboards section of your Grafana instance.
```
```suggestion
To try it out, go to the **Dashboards** section of your Grafana instance.
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+<!-- Nathan Marrs -->
+<!-- Cloud -->
+
+You can now add buttons to your Canvas visualizations. Buttons can be configured to call an API endpoint. This pushes Grafana's capabilities to new heights, allowing you to create interactive dashboards that can be used to control external systems.
```
```suggestion
You can now add buttons to your canvas visualizations. Buttons can be configured to call an API endpoint. This pushes Grafana's capabilities to new heights, allowing you to create interactive dashboards that can be used to control external systems.
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+{{< video-embed src="/media/docs/grafana/2023-20-10-Canvas-Button-Element-Enablement-Video.mp4" max-width="750px" caption="Canvas button element demo" >}}
+
+### Zoom in on the Y-axis of the Time Series and Candlestick panels
```
```suggestion
### Zoom in on the y-axis of the Time series and Candlestick visualizations
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+### Zoom in on the Y-axis of the Time Series and Candlestick panels
+
+_Generally available in Grafana Open Source and Enterprise_
```
```suggestion
_Generally available in all editions of Grafana_
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+Now, you can automatically calculate the min or max of each visualized field based on the lowest and highest value of the individual field. This setting is available in the standard options of most visualizations.
+
+{{< figure src="/media/docs/grafana/panels-visualizations/localminmax.png" max-width="300px" caption="Stat panel visualization with min/max calculated per field" >}}
```
```suggestion
{{< figure src="/media/docs/grafana/panels-visualizations/localminmax.png" max-width="300px" caption="Stat visualization with min/max calculated per field" >}}
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+### Data visualization quality of life improvements
+
+_Generally available in Grafana Open Source and Enterprise_
```
```suggestion
_Generally available in all editions of Grafana_
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+<!-- Joey Tawadrous, Jen Villa -->
+
+_Experimental in Grafana Open Source and Enterprise_
```
```suggestion
_Experimental in all editions of Grafana_
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+{{< figure src="/media/docs/tempo/multiple-spansets-per-trace-10-2.png" max-width="750px" caption="Multiple spansets per trace" >}}
+
+#### Create Query-type template variables for the Tempo data source
```
```suggestion
#### Create query-type template variables for the Tempo data source
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+_Generally available in Grafana Enterprise and Grafana Cloud Advanced_
+
+The SAP HANA® data source now supports tenant database connections by using the database name and/or instance number, which are less likely to change than the port for your database. For more information, refer to our [SAP HANA® configuration documentation](/docs/plugins/grafana-saphana-datasource/latest/#configuration).
```
```suggestion
The SAP HANA® data source now supports tenant database connections by using the database name and/or instance number. This is helpful because these are less likely to change than the port for your database. For more information, refer to our [SAP HANA® configuration documentation](/docs/plugins/grafana-saphana-datasource/latest/#configuration).
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+{{< figure src="/media/docs/grafana/transformations/unary-operation.png" >}}
+
+**Row index** can now show the index as a percentage.
```
```suggestion
Also, **Row index** can now show the index as a percentage.
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+<!-- Alexander Weaver -->
+
+_Generally available in all editions of Grafana_
```
```suggestion
_Generally available in Grafana Open Source and Enterprise_
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+### New Service Account permissions
+
+[Service accounts](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/service-accounts/) allow you to create tokens to access Grafana's API and dashboards. Service accounts are a powerful tool for authenticating with Grafana's API and accessing data sources. However, without proper access controls, service accounts can pose a security risk to your Grafana instance. In Grafana 10.2, ...
```
```suggestion
[Service accounts](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/service-accounts/) allow you to create tokens to access Grafana's API and dashboards. Service accounts are a powerful tool for authenticating with Grafana's API and accessing data sources. However, without proper access controls, service accounts can pose a security risk to your Grafana instance. In Grafana 10.2, we've added new tools to limit service accounts to just the resources they need to access.
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+_Generally available in Grafana Enterprise_
+
+Recorded queries provide a way to take a _static_ number, (say, the number of GitHub issues open at a given time, or the number of rows in a database table) and record it periodically as a Prometheus metric. This is great for tracking numbers over time for quick querying later. Previously, Recorded Queries were limited to a single series, so you needed to narrow your query down to a single number ...
```
```suggestion
Recorded queries provide a way to take a _static_ number, (for example, the number of GitHub issues open at a given time, or the number of rows in a database table) and record it periodically as a Prometheus metric. This is great for tracking numbers over time for quick querying later. Previously, recorded queries were limited to a single series, so you needed to narrow your query down to a single number in order to record it. Now, you can record multiple metrics with a single recorded query, which makes them more powerful _and_ easier to create and manage.
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,476 @@
+
+<!-- Kyle Cunningham -->
+
+_Available in public preview in Grafana Open Source and Enterprise_
```
```suggestion
_Available in public preview in all editions of Grafana_
```

imatwawana committed:
Copy edits

shelldandy committed:
update enterprise ds availability

mjseaman committed:
Update whats-new-in-v10-2.md

imatwawana committed:
Fixed style issues and availability note

imatwawana committed:
Added missing Alerting items

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,494 @@
+
+_Generally available in all editions of Grafana_
+
+This feature provides a way to export Alerting resources such as rules, contact points, and notification policies as Terraform resources. A new "Modify export" mode for alert rules provides a convenient way of editing provisioned alert rules and exporting the modified version.
```
```suggestion
This feature provides a way to export alerting resources such as rules, contact points, and notification policies as Terraform resources. A new "Modify export" mode for alert rules provides a convenient way of editing provisioned alert rules and exporting the modified version.
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,494 @@
+
+<!-- Kyle Brandt, Observability Metrics -->
+
+_Generally available in Grafana Enterprise_
```
```suggestion
_Generally available in Grafana Enterprise and Grafana Cloud_
```

imatwawana committed:
Updated availability note

imatwawana committed:
Fixed styling

imatwawana committed:
Reverted what's new link to relref

mjseaman committed:
update availability of No Basic Role

I consulted with Eric before making this change.

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,494 @@
+
+Edit provisioned alerting resources using file provisioning or Terraform directly from within the Grafana UI.
+
+### Recovery thresholds for alerts
+
+<!-- George Robinson -->
+
+_Generally available in all editions of Grafana_
+
+To reduce the noise of flapping alerts, you can set a recovery threshold different from the alert threshold.
+
+Flapping alerts occur when a metric hovers around the alert threshold condition and they may lead to frequent state changes, resulting in too many notifications being generated.
+
```
```suggestion
```

imatwawana committed:
Removed item not going out in this release

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,484 @@
+
+We've added support for the Microsoft Teams contact points when using an external Alertmanager.
+
+### Edit file-provisioned and Terraform-provisioned alerting resources
+
+<!-- George Robinson -->
+
+_Generally available in all editions of Grafana_
+
+Edit provisioned alerting resources using file provisioning or Terraform directly from within the Grafana UI.
+
```
```suggestion
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-2.md:
```diff
@@ -0,0 +1,484 @@
+
+To learn more, refer to the [Grafana OnCall integration for Alerting documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/alerting/alerting-rules/manage-contact-points/configure-oncall/).
+
+### Export alert rules and notification resources to Terraform
+
+<!-- Yuri Tseretyan -->
+
+_Generally available in all editions of Grafana_
+
+This feature provides a way to export alerting resources such as rules, contact points, and notification policies as Terraform resources. A new "Modify export" mode for alert rules provides a convenient way of editing provisioned alert rules and exporting the modified version.
```
```suggestion
### Export alerting resources to Terraform

<!-- Yuri Tseretyan -->

_Generally available in all editions of Grafana_

Export your alerting resources, such as alert rules, contact points, and notification policies as Terraform resources. A new “Modify export” mode for alert rules enables you to edit provisioned alert rules and export a modified version. 
```

imatwawana committed:
Update Alerting items

imatwawana committed:
Fixed linting
